content/en/docs/architecture/step-registry.md

```diff
@@ -390,8 +390,8 @@ Other files that are allowed in the step registry but are not used for testing a
 
 ## Parameters
 
-Steps, chains, and workflows can declare parameters in their `env` section. These can then be set to different values to generate tests
-that have small variations between them. For example:
+Steps, chains, and workflows can declare parameters they expect to consume in their `env` section. These can then be set to different values to generate tests
```

Comment:
Workflows do not consume parameters, they set values for them.

Reply:
Sigh